src/GCodes/GCodeException.h

```diff
@@ -16,32 +16,45 @@ namespace StackUsage
 	constexpr uint32_t Margin = 300;					// the margin we allow for calls to non-recursive functions that can throw
 }
 
+enum class GCodeExceptionSource : uint8_t
+{
+	OTHER,
```

Comment:
Please don't declare the enum constants using ALL CAPS. We use all caps only for macros.